modules/apps/commerce/commerce-channel-web/src/main/resources/META-INF/resources/commerce_channel/general.jsp

```diff
@@ -77,12 +80,29 @@ Map<String, String> contextParams = HashMapBuilder.<String, String>put(
 					%>
 
 				</aui:select>
+
+				<c:if test='<%= FeatureFlagManagerUtil.isEnabled("COMMERCE-10890") %>'>
+					<aui:select disabled="<%= !commerceChannelDisplayContext.hasManageLinkSupplierPermission() %>" label="link-channel-to-a-supplier" name="accountEntryId" showEmptyOption="<%= true %>">
```

Comment:
so this is always visible, right?

Reply:
Always visible (if feature flag is enabled) but disabled if you don't have permission to edit it